packages/arb-bridge-eth/contracts/rollup/facets/RollupUser.sol

```diff
@@ -591,14 +599,18 @@ contract RollupUserFacet is AbsRollupUserFacet {
         external
         override
         onlyValidator
-        whenNotPaused
+        whenInShutdownModeOrNotPaused
         returns (uint256)
     {
         uint256 amount = withdrawFunds(msg.sender);
         // This is safe because it occurs after all checks and effects
         destination.transfer(amount);
         return amount;
     }
+
+    function isNitroReady() external pure returns (uint8) {
+        return uint8(0xa4b1);
+    }
```

Comment:
Very minor but this should probably be on `AbsRollupUserFacet` instead of `RollupUserFacet` which is ETH-only.